### PR TITLE
Different datatype for round number among different tables - Closes #1647

### LINKED ISSUE #1647: Different datatype for round number among different tables
### Expected behavior
`round` number among different database tables should have same data type 

### Actual behavior
- In table `mem_round` the column `round` have `BIGINT` datatype
- In table `rounds_rewards` the column `round` have `INT` datatype

That cause to manipulate `round` number differently in the code when fetched from different tables. Also that may cause problems when we have larger round numbers. 

### Which version(s) does this affect? (Environment, OS, etc...)
`1.0.0`

### PR DESCRIPTION
**Pull request by 4miners**
### What was the problem?
Inconsistent `round` data type among different tables.
### How did I fix it?
Change `round` data type to `INT` for `mem_round` table.
### How to test it?
Run test suite.
### Review checklist

* The PR solves #1647
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: db/sql/migrations/updates/20180626001337_alter_mem_round.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ */
+
+
+/*
+  DESCRIPTION: Alter 'mem_round' table - change data type of 'round' from BIGINT to INT.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE mem_round ALTER COLUMN round TYPE INT;
```
Comment (nazarhussain) [inline db/sql/migrations/updates/20180626001337_alter_mem_round.sql]: Please update the migration file name to more meaning full name e.g. `alter_round_in_mem_rounds` or `change_round_type_in_mem_rounds`.

File: db/sql/migrations/memory_tables.sql [hunk 2]
```diff
@@ -55,7 +55,7 @@ CREATE TABLE IF NOT EXISTS "mem_round"(
   "address" VARCHAR(22),
   "amount" BIGINT,
   "delegate" VARCHAR(64),
-  "round" BIGINT
+  "round" INT
 );
 
 CREATE INDEX IF NOT EXISTS "mem_round_address" ON "mem_round"("address");
```
Comment (nazarhussain) [inline db/sql/migrations/memory_tables.sql]: This file is not used anywhere. Please remove it in current PR or create another issue for it, so we shouldn't forget.

**Comment by 4miners** [comment]
@nazarhussain Resolved, thanks for review.

**Comment by MaciejBaj** [comment]
@4miners please check the tests.

**Comment by 4miners** [comment]
@MaciejBaj That PR is 7 days old, tests started to fail yesterday because something is broken in 1.1 branch currently - it's not because of changes in this PR.

**Pull request merged**